beacon_node/beacon_chain/src/blob_verification.rs

```diff
@@ -145,28 +145,19 @@ pub enum GossipBlobError<E: EthSpec> {
     NotFinalizedDescendant { block_parent_root: Hash256 },
 }
 
-impl<E: EthSpec> std::fmt::Display for GossipBlobError<E> {
+impl std::fmt::Display for GossipBlobError {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        match self {
-            GossipBlobError::BlobParentUnknown(blob_sidecar) => {
-                write!(
-                    f,
-                    "BlobParentUnknown(parent_root:{})",
-                    blob_sidecar.block_parent_root()
-                )
-            }
-            other => write!(f, "{:?}", other),
-        }
+        write!(f, "{:?}", self)
```

Comment:
Kept this implementation of Display as some release tests assert the full error description. I'm not sure of which consumers expect the full error details in Display formatting. If we switch to `strum::Display` we may lose information in some error logs throughout the codebase. Thoughts?